python/tvm/auto_scheduler/measure.py

```diff
@@ -718,11 +709,18 @@ def local_builder_build(inputs, timeout, n_parallel, build_func="default", verbo
     for res in tuple_res:
         if res.status == StatusKind.COMPLETE:
             results.append(BuildResult(*res.value))
-        else:
-            assert res.status == StatusKind.TIMEOUT
+        elif res.status == StatusKind.TIMEOUT:
             if verbose >= 1:
                 print(".T", end="", flush=True)  # Build timeout
             results.append(BuildResult(None, [], MeasureErrorNo.BUILD_TIMEOUT, None, timeout))
+        elif res.status == StatusKind.EXCEPTION:
+            if verbose >= 1:
+                print(".E", end="", flush=True)  # Build error
+            results.append(
+                BuildResult(None, [], MeasureErrorNo.COMPILE_HOST, make_traceback_info(), timeout)
```

Comment:
`make_traceback_info` on the host side won't reflect the actual error that happens on the worker. Let's use the exception propagated from `res.value` instead. Don't forget the stringify the exception :-)

Reply:
done